django/middleware/csrf.py

```diff
@@ -207,9 +212,14 @@ def _reject(self, request, reason):
         return response
 
     def _get_token(self, request):
+        """
+        Return the CSRF secret originally associated with the request, or None
+        if it didn't have one. Raises InvalidTokenFormat if sessions aren't
```

Comment:
Shouldn't this say `ImproperlyConfigured`? Also from the comment alone it seems to suggest that this would always raise if sessions are not enabled (but it only raises if `CSRF_USE_SESSIONS`

Reply:
I pushed a new version that improves the wording of the docstring and hopefully removes any confusion. (I don't think it's worth calling out the `ImproperlyConfigured` exception though since, unlike `InvalidTokenFormat`, it's not something the calling code can handle.)